Cartfile.resolved

```diff
@@ -1,5 +1,5 @@
 github "Alamofire/Alamofire" "5.0.0-rc.2"
```

Comment:
hmm, I wonder why the Alamofire version didn't get bumped to "5.0" 🤔

Reply:
Someone probably forgot that for Cartage?